File: src/gestorAplicacion/tareas/Actividad.java
package gestorAplicacion.tareas;

import gestorAplicacion.horario.Asignatura;

public class Actividad extends Tarea{
	
	String descripcion;
	
	public Actividad(String titulo, Asignatura asignatura, String fechaEntrega) {
		super(titulo, asignatura, fechaEntrega);
	}


	
	@Override
	void cambiarEstado() {
		this.estado = Estado.completada;
 	}
	
	
}

File: python/tareas/actividad.py
from tareas.proyecto import proyecto
from tareas.tarea import tarea

#Objetos de tipo tarea y proyecto pero que a diferencia de proyecto solo admite objetos que solo sean tipo tarea
class actividad(proyecto, tarea):
    def __init__(self, titulo, asignatura, fechaDeEntrega, estado):
        self.descripcion
        super().__init__(titulo, asignatura, fechaDeEntrega, estado)
    
    


    